server/telegram.go

```diff
@@ -158,3 +166,49 @@ func generateKeyboardForMemoID(id int) [][]telegram.InlineKeyboardButton {
 
 	return [][]telegram.InlineKeyboardButton{buttons}
 }
+
+func ConvertToMarkdown(text string, messageEntities []telegram.MessageEntity) string {
```

Comment:
If all Telegram message entity could convert or compatible with `Markdown`?

If this can be done, just add all entity type, else leave it unsupported is better than support with exceptions.

Reply:
The entity specifies the type of formatting: emoji, spoiler, url, mention, classic bold, italic, etc.

I have added processing of those that your markdown supports. Not in markdown right now. for example, underline.

For specific entities there is no analogue in markdown and not all of them makes sense to try to format somehow. Therefore, they are ignored.